smalltalksrc/Slang/CSlangPrettyPrinter.class.st

```diff
@@ -206,6 +220,15 @@ CSlangPrettyPrinter >> visitIfStatement: anIf [
 		e acceptVisitor: self ]
 ]
 
+{ #category : #generated }
+CSlangPrettyPrinter >> visitIncrement: anIncrementNode [
+	| visit |
+	visit := [ anIncrementNode object acceptVisitor: self ].
+	anIncrementNode prefix ifFalse: visit.
```

Comment:
So this is the same for pre and post increment?

Reply:
Yes, is this a problem ?